lib/localeapp/cli/install.rb

```diff
@@ -118,7 +134,7 @@ def validate_key
           if key.nil?
             @output.puts "ERROR: No api key found in heroku config, have you installed the localeapp addon?"
             return
-          else
+          elsif not self.dotenv
```

Comment:
The code never more exists, has been dropped with detection of `.env`'s existence.